tests/BasicTest.hack

```diff
@@ -57,6 +57,16 @@ final class BasicTest extends PlaygroundTest {
                 dict[],
                 dict['pokeUser' => dict['id' => 2]],
             ),
+            'enum arguments' => tuple(
+                'query { takes_favorite_color(favorite_color: red) }',
```

Comment:
This should accept `RED` but not `red`. Maybe we could change the `FavoriteColor` enum to be an int enum to make this distinction clearer:

```Hack
enum FavoriteColor: int {
  RED = 1;
  BLUE = 2;
}
```

This also makes it much more obvious why accepting the const names is better than using the internal values; we wouldn't want GraphQL clients to have to write `favorite_color: 2`